### ISSUE TITLE
Clearer instructions

### ISSUE THREAD

**Issue by SassNinja** [sentientmachine/erics_vim_syntax_and_color_highlighting#5]
@sentientmachine I came here from your post on stackoverflow
https://stackoverflow.com/questions/11272501/enable-vim-syntax-highlighting-by-default#answer-20795372

I'm trying to achieve the same highlighting for my files, but unfortunately there are no clear instructions how to use it.
I've copied `colors` and `syntax` folder to my home `.vim` directory and defined the following in my `.vimrc` – but still not working
```bash
execute pathogen#infect()
syntax on
filetype plugin indent on
```
What you've written on stackoverflow
> Then configure some extra commands in your /home/yourusername/.vimrc and you should get what I get.

doesn't help a lot because the commands are not shown anywhere.
Would you please tell me what to do (and ideally add it to the README)?

**Comment by sentientmachine** [comment]
I've added the reason why I can't update this here.  This project has become a single point of reference for me between Ubuntu, Fedora, Arch and Gentoo which all require special handholding.  For me to unify it would take many hours of extra effort to hold your hand and would be no return on investment for me.

Like I said in the readme, read the code, all the clues and pointers are there, you'll encounter errors because the version of linux you're using is different than mine, because the libraries this stuff relies on haven't been installed, also because you don't have special programs of intellectual property I have that undergird certain elite functions.  

It's a rube goldberg machine, and this project sprawls over the entire disk and I can't upload it all, it's on a private repo.  All the clues you need are here though.  Each line has a purpose, or at least used to as a vestigial part.  It truly evolves over time.  

The images I've uploaded here are are but a simple taste of what I have going.  If you saw my context sensitive autocompletion, syntax aware help, auto highlighting of errors, you would be blown away.  

Vim is more powerful as an IDE than visual studio, keep going and keep hacking.  Teach yourself how to fish, and it's so much more gratifying than me just handing the entire thing to you, you couldn't handle the raw power anyway, since the first sign of it breaking, you wouldn't know where to begin to autocorrect the autocorrector.

**Comment by SassNinja** [comment]
@sentientmachine why have you edited your post and removed the note regarding the forks?
Following the instructions of one fork (which you've removed in the main repo) made it possible for me to get the highlighting working.

I understand you don't wanna hold peoples hand and advice people to learn more themselves. It's comprehensible you don't wanna spent much time in creating instructions for all systems.

But removing a working instruction to force people to go the hard way is completely ridiculous imo!

> If you saw my context sensitive autocompletion, syntax aware help, auto highlighting of errors, you would be blown away.

Sounds cool but would definitely not blow me away – sorry. My IDE (visual studio code) provides the same features (probably even better) and didn't take much time to setup. The reason I'm trying to get more familiar with the terminal is I'd like to be able to work more on servers and may be faster with small code changes.

However the forks (you mentioned earlier) helped me to get it running. I'm using Mac OS.
In case someone else is facing the same issue and finds this post I'll repeat the steps:

1. clone the repo into the `.vim` folder (home directory)
2. `mv erics_vim_syntax_and_color_highlighting/* .`
3. `rm -rf erics_vim_syntax_and_color_highlighting`

Then edit the `.vimrc` file and add

```
"The following three lines map Ctrl+s to save in vi.  You can comment 
"these out, it has nothing to do with syntax highlighting or colors.
:nmap <c-s> :w<CR>
:imap <c-s> <Esc>:w<CR>a
:imap <c-s> <Esc><c-s>


set background=dark
set hlsearch
set nu
set smartindent
set tabstop=4
set shiftwidth=4
set expandtab
filetype on
filetype plugin indent on
syntax on

au BufReadPost *.twig colorscheme koehler 
au BufReadPost *.css colorscheme slate 
au BufReadPost *.js colorscheme elflord 
au BufReadPost *.py colorscheme molokai
au BufReadPost *.html colorscheme monokai
au BufReadPost *.java colorscheme monokai
au BufReadPost *.php colorscheme monokai
```

**Comment by sentientmachine** [comment]
That's correct and part of it to assert the colorschemes.  But for some of the other tools to work you have to assert filetype as well.   But of course those extra things aren't going to work anyway because I haven't mentioned the plugins and my intellectual property that undergird them.

I don't know why I removed them, I guess it's a philosophical breakdown.  I guess it's the same reason a capitalist walks into a forest and finds a goose that lays golden eggs, and rather than making a road map for everyone else to have one too, say: "There's no such thing... Go away".  

If these things are in your destiny, then lack of instructions won't prevent you from also making your own.
